modules/backend/widgets/Filter.php

```diff
@@ -305,7 +345,8 @@ protected function getOptionsFromModel($scope, $searchQuery = null)
         /*
          * Extensibility
          */
-        $this->fireSystemEvent('backend.filter.extendQuery', [$query, $scope]);
```

Comment:
Why did you replace this with the split version? `fireSystemEvent()` does exactly the same thing as calling both global and local events separately here.